tests/js/spec/components/group/__snapshots__/externalIssueForm.spec.jsx.snap

```diff
@@ -760,18 +760,6 @@ exports[`ExternalIssueForm link renders 1`] = `
                                         onCloseResetsInput={false}
                                         onKeyDown={[Function]}
                                         onSelectResetsInput={false}
-                                        options={
-                                          Array [
-                                            Object {
-                                              "label": "test",
-                                              "value": "scefali/test",
```

Comment:
Hmm I'll need to manually test these to make sure I didn't break anything

Reply:
@scefali and I poked through this in staging and seems to work, this is expected because previously SelectField --> SelectControl would parse `choices` and apply `options`, but in this PR we've changed it so that only `SelectControl` does it